pkg/codegen/internal/test/sdk_driver.go

```diff
@@ -44,85 +44,109 @@ var sdkTests = []sdkTest{
 		Directory:        "nested-module-thirdparty",
 		Description:      "Third-party nested module",
 		SkipCompileCheck: codegen.NewStringSet(dotnet, nodejs),
+		Skip:             codegen.NewStringSet("python/test"),
 	},
 	{
 		Directory:   "plain-schema-gh6957",
 		Description: "Repro for #6957",
+		Skip:        codegen.NewStringSet("python/test"),
 	},
 	{
 		Directory:   "resource-args-python-case-insensitive",
 		Description: "Resource args with same named resource and type case insensitive",
+		Skip:        codegen.NewStringSet("python/test"),
 	},
 	{
 		Directory:   "resource-args-python",
 		Description: "Resource args with same named resource and type",
+		Skip:        codegen.NewStringSet("python/test"),
 	},
 	{
 		Directory:   "simple-enum-schema",
 		Description: "Simple schema with enum types",
+		Skip:        codegen.NewStringSet("python/test"),
 	},
 	{
 		Directory:   "simple-plain-schema",
 		Description: "Simple schema with plain properties",
+		Skip:        codegen.NewStringSet("python/test"),
 	},
 	{
 		Directory:   "simple-plain-schema-with-root-package",
 		Description: "Simple schema with root package set",
+		Skip:        codegen.NewStringSet("python/test"),
 	},
 	{
 		Directory:   "simple-resource-schema",
 		Description: "Simple schema with local resource properties",
+		Skip:        codegen.NewStringSet("python/test"),
 	},
 	{
 		Directory:   "simple-resource-schema-custom-pypackage-name",
 		Description: "Simple schema with local resource properties and custom Python package name",
+		Skip:        codegen.NewStringSet("python/test"),
 	},
 	{
 		Directory:        "simple-methods-schema",
 		Description:      "Simple schema with methods",
 		SkipCompileCheck: codegen.NewStringSet(nodejs, dotnet, golang),
+		Skip:             codegen.NewStringSet("python/test"),
 	},
 	{
 		Directory:   "simple-yaml-schema",
 		Description: "Simple schema encoded using YAML",
+		Skip:        codegen.NewStringSet("python/test"),
 	},
 	{
 		Directory:        "provider-config-schema",
 		Description:      "Simple provider config schema",
 		SkipCompileCheck: codegen.NewStringSet(dotnet),
+		Skip:             codegen.NewStringSet("python/test"),
 	},
 	{
 		Directory:        "replace-on-change",
 		Description:      "Simple use of replaceOnChange in schema",
 		SkipCompileCheck: codegen.NewStringSet(golang),
+		Skip:             codegen.NewStringSet("python/test"),
 	},
 	{
 		Directory:        "resource-property-overlap",
 		Description:      "A resource with the same name as it's property",
 		SkipCompileCheck: codegen.NewStringSet(dotnet, nodejs),
+		Skip:             codegen.NewStringSet("python/test"),
 	},
 	{
 		Directory:   "hyphen-url",
 		Description: "A resource url with a hyphen in it's path",
+		Skip:        codegen.NewStringSet("python/test"),
+	},
+	{
+		Directory:        "output-funcs",
+		Description:      "Tests targeting the $fn_output helper code generation feature",
+		SkipCompileCheck: codegen.NewStringSet(dotnet),
```

Comment:
I intend to fix .NET compilation rebasing https://github.com/pulumi/pulumi/pull/7899 onto this work shortly after it's merged.